### PR TITLE
The Unit of a Test blog entry

### PR DESCRIPTION
**Pull request by eamelink**

### CODE REVIEW

File: posts/2024-11-15-the-unit-of-a-test.adoc [hunk 1]
```diff
@@ -0,0 +1,140 @@
+= The Unit of a test
+eamelink
+v1.0, 2024-11-15
+:title: The Unit of a test
+:lang: en
+:tags: [java, testing, quality]
```
Comment (nikashamova) [inline posts/2024-11-15-the-unit-of-a-test.adoc:6]: Can you please add `java-vs-world` and `java-over-java` tags?

File: posts/2024-11-15-the-unit-of-a-test.adoc [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+----
+public class UserService {
+
+	private final PeopleRepository peopleRepository;
+
+	public UserService(PeopleRepository peopleRepository) {
+		this.peopleRepository = peopleRepository;
+	}
+
+	public getUserById(String id) {
```
Comment (nikashamova) [inline posts/2024-11-15-the-unit-of-a-test.adoc:24]: I think return type is missing

File: posts/2024-11-15-the-unit-of-a-test.adoc [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+class UserServiceTest {
+
+	@Mock
+	UserMapper userMapper;
+
+	@Mock
+	PeopleRepository peopleRepository;
+
+	@Test
+	testGetById() {
```
Comment (nikashamova) [inline posts/2024-11-15-the-unit-of-a-test.adoc:83]: I think also `void` here as a return type

File: posts/2024-11-15-the-unit-of-a-test.adoc [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+	private final PeopleRepository peopleRepository;
+	private final UserMapper userMapper;
+
+	public UserService(PeopleRepository peopleRepository, UserMapper userMapper) {
+		this.peopleRepository = peopleRepository;
+		this.userMapper = userMapper;
+	}
+
+	public getUserById(String id) {
```
Comment (nikashamova) [inline posts/2024-11-15-the-unit-of-a-test.adoc:54]: Same here 🙈

**Comment by tesfayenati** [comment]
It looks good except Nika's comments :-)

**Review by j-madden — APPROVED** [review]

**Review by tesfayenati — APPROVED** [review]

**Pull request merged**